.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+on: push
+
+env:
+  NODE_VERSION: "16"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+      - run: TZ=Europe/Berlin yarn --prefer-offline
+      - run: yarn lint
```

Comment:
I think installation of node modules is missing?
```
      - run: yarn --prefer-offline
```

Reply:
Just curious, is there a difference when setting the timezone in the `run: yarn` command instead of the usual:
```yml
env:
  TZ: Europe/Berlin
```

Reply:
@David3141 
```
- run: TZ=Europe/Berlin yarn --prefer-offline
```

This installs the node modules with a TZ env export

@Finnegan5 
No idea, I'll change it and see if it still works

Reply:
Still seems to work with `TZ` in `env` 👍

Reply:
Oh sorry, didn't notice that